texthero/preprocessing.py

```diff
@@ -614,3 +616,54 @@ def remove_urls(s: pd.Series) -> pd.Series:
     """
 
     return replace_urls(s, " ")
+
+
+def correct_mistakes(s: pd.Series, fix_spacing=False) -> pd.Series:
+    r""" Correct spelling mistakes
+
+    `correct_mistakes` finds all spelling mistakes from the given Pandas Series and replace them with suggested words.
+
+    When `fix_spacing` is set to ´True´, word_segmentation is applied.
+
+    Parameters
+    ----------
+    s : Pandas Series
+    fix_spacing : bool
+        When set to True, word_segmentation is applied, spacing is fixed in noisy text. Only use it if your data is similar to 'thequickbrownfoxjumpsoverthelazydog' as it slows performance
```

Comment:
it would be great, if the docstring length is no longer than 88 chars

Reply:
Ok will split it by new lines